launch.rb

```diff
@@ -149,7 +112,21 @@ def self.prepare_server
       FastlaneCI::FastlaneApp.use(FastlaneCI::BuildController)
     end
 
+    def self.run_github_workers
+      return if first_time_user?
+
+      launch_workers
+
+      github_projects = Services.config_service.projects(provider_credential: self.provider_credential)
+      github_service = FastlaneCI::GitHubService.new(provider_credential: self.provider_credential)
+
+      run_pending_github_builds(projects: github_projects, github_service: github_service)
+      enqueue_builds_for_open_github_prs_with_no_status(projects: github_projects, github_service: github_service)
+    end
+
     def self.launch_workers
+      return if first_time_user?
```

Comment:
Ahh, yeah - that's a good point. The reason I had the guard was in case someone wants to call `Launch.launch_workers` outside of the context of `Launch.run_github_workers`.

Do you think it's worth having the guard statement, or should I just have the one guard in `Launch.run_github_workers`?